frontend/src/views/components/AccountInput.tsx

```diff
@@ -67,49 +53,38 @@ export const AccountInput = ({
     formState: { errors: errorsUpdatedUser },
   } = useForm<FormData>();
 
-  const account = useSelector(selectUser);
-
   const handleEdit = () => {
     setEditStatus(true);
   };
 
   const handleSave = async (formData: FormData) => {
-    const { firstName, lastName, email } = formData;
-
-    const updatedMetaData = {
-      firstName: firstName,
-      lastName: lastName,
-    };
-
-    const { data, error } = await supabase.auth.updateUser({
-      email: email,
-      data: updatedMetaData,
-    });
-    if (error) {
+    const { isError, createdUser } = await UserService.updateAuthUser(formData);
+    if (isError) {
```

Comment:
At this point, your code looks good! However, as next steps, you can learn early return practice.
https://methodpoet.com/return-early-or-if-statement/#What_is_an_early_return

By using this practice, this code will become readable.

Reply:
I got it. I remove the nest and changed it like below.
```js
    const { isError, createdUser } = await UserService.updateAuthUser(formData);
    if (isError) {
      return setUpdateUserInfoMessage({
        isError: true,
        message: addNewLinesAfterPunctuation(ERROR_CHANGE_ACCOUNT_INFO),
      });
    }

    const { isError: isErrorForUpdateUser } = await UserService.updateUser(
      createdUser!
    );
```

Reply:
Thank you! It becomes more readable!